workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -127,7 +135,8 @@ process quant_fry{
      --tg-map ${tx2gene} \
      --output-dir ${run_dir} \
      -r ${params.resolution} \
-     -t ${task.cpus}
+     -t ${task.cpus} \
+     ${params.resolution == 'cr-like' ? '--use-mtx' : ''}
```

Comment:
We are using mtx here so that we can use the alevin USA-reading function, correct? I feel like it is odd to tie this to `cr-like` as we could use that for the "normal" index as well, not just for `splici`.

Reply:
It looks also like this might be moot in the future? From https://combine-lab.github.io/alevin-fry-tutorials/2021/improving-txome-specificity/: 
> While presented as a stand-alone function above, we plan to add the functionality for loading and processing quantification estimates generated in USA-mode into fishpond / tximeta in the current development cycle.

Reply:
Yes, I only added that in for right now and when I actually get a chance to play around with importing the quants files then we may be able to alter this. But my reasoning for tying it to `cr-like` is because that is the only resolution that has compatibility to be used with the USA mode and therefore would benefit from being processed using their USA functions. 

Are you saying that you think we should be using the `--use-mtx` flag for more than just the `cr-like` resolution? Or that we should be restricting it to `cr-like` and `splici` so it isn't also used for the cDNA index with `cr-like`? 

I also would love to be able to take it out in the future and have them all have the same output format, I just didn't want to do that without having a chance to actually look at both outputs.

Reply:
> Are you saying that you think we should be using the --use-mtx flag for more than just the cr-like resolution? Or that we should be restricting it to cr-like and splici so it isn't also used for the cDNA index with cr-like?

I think I was saying the latter, as I am pretty sure that we would be fine to read in cDNA/cr-like data with current scripts. 

I went back to read the example more carefully, and my understanding is that this should probably be conditional on the `tx2gene` file rather than the resolution mode alone? If the `tx2gene` file is three columns, that is the trigger for USA mode (I missed that point before), which then requires the mtx output so we can read the matrix (now 3x times longer) "manually" rather than with `tximport`/`tximeta`.  Since the use of 3col is contingent on `cr-like` and `splici`, I would expect that to be condition here.

That said, if we have to use the custom read function for mtx data sometimes, we could modify our downstream scripts to use that function (and mtx) everywhere.  But we probably don't want to recalculate things for benchmarking, and long term (hopefully sooner rather than later) I might rather we were using tximeta/tximport...